tox.ini

```diff
@@ -0,0 +1,29 @@
+# this template can be adjusted as needed for your CI environment. the paths in the `commands` of each tox env linter can be changed if the tox file sits in a different location than expected in this template
+[tox]
+envlist = ruff, darglint, pylint
+requires = 
+    ruff
+    darglint
+    pylint
+
+[testenv:ruff]
+deps = ruff
+commands = ruff check --select ALL --ignore INP001 -q /extensions/eda/plugins
+
+
+[testenv:darglint]
+deps = darglint
+commands = darglint -s numpy -z full /extensions/eda/plugins
+
+# depending on what kind of plugins you have, remove the line you don't need (i.e event_sources or event_filters, remove the pylint call for the other one)
+# depending on how your collection and repo is structured, you may need to change the path to each type of plugin
+# if pylint warns about missing __init__.py files in directories, there's no need to include them if you ensure that the paths in the below pylint `commands` point directly to the *.py files under the event_source/ and event_filter/ dirs, as shown in the template path here
+[testenv:pylint-event-source]
+deps = pylint
+commands = 
+    pylint /extensions/eda/plugins/event_source/*.py --output-format=parseable -sn --disable R0801
+
+[testenv:pylint-event-filter]
+deps = pylint
+commands = 
```

Comment:
```suggestion
commands = pylint /extensions/eda/plugins/event_filter/*.py --output-format=parseable -sn --disable R0801
```

Reply:
Placing it on the same line as command. Was causing some issues when I tested it with it on a line below.